sql/monthly-revenue-query.sql

```diff
@@ -179,104 +204,105 @@ SELECT
-    SUM(recurringMonthlyNew * "isUser") as "numRecurringMonthlyNewDonationsFromUsers",
-    SUM(recurringMonthlyNew * "isOrg") as "numRecurringMonthlyNewDonationsFromOrgs",
+    SUM(recurringMonthlyNew * (isNotRefund + isRefund)) as "numRecurringMonthlyNewDonations",
+    SUM(recurringMonthlyNew * (isNotRefund + isRefund) * "isUser") as "numRecurringMonthlyNewDonationsFromUsers",
+    SUM(recurringMonthlyNew * (isNotRefund + isRefund) * "isOrg") as "numRecurringMonthlyNewDonationsFromOrgs",
 
     /* annually */
-    SUM(recurringAnnuallyTotal) as "numRecurringAnnualDonations",
-    SUM(recurringAnnuallyTotal * "isUser") as "numRecurringAnnuallyTotalDonationsFromUsers",
-    SUM(recurringAnnuallyTotal * "isOrg") as "numRecurringAnnuallyTotalDonationsFromOrgs",
+    SUM(recurringAnnuallyTotal * (isNotRefund + isRefund)) as "numRecurringAnnualDonations",
+    SUM(recurringAnnuallyTotal * (isNotRefund + isRefund) * "isUser") as "numRecurringAnnuallyTotalDonationsFromUsers",
+    SUM(recurringAnnuallyTotal * (isNotRefund + isRefund) * "isOrg") as "numRecurringAnnuallyTotalDonationsFromOrgs",
 
-    SUM(recurringAnnuallyOld) as "numRecurringAnnuallyOldDonations",
-    SUM(recurringAnnuallyOld * "isUser") as "numRecurringAnnuallyOldDonationsFromUsers",
-    SUM(recurringAnnuallyOld * "isOrg") as "numRecurringAnnuallyOldDonationsFromOrgs",
+    SUM(recurringAnnuallyOld * isNotRefund) as "numRecurringAnnuallyOldDonations",
+    SUM(recurringAnnuallyOld * isNotRefund * "isUser") as "numRecurringAnnuallyOldDonationsFromUsers",
+    SUM(recurringAnnuallyOld * isNotRefund * "isOrg") as "numRecurringAnnuallyOldDonationsFromOrgs",
 
-    SUM(recurringAnnuallyNew) as "numRecurringAnnuallyNewDonations",
-    SUM(recurringAnnuallyNew * "isUser") as "numRecurringAnnuallyNewDonationsFromUsers",
-    SUM(recurringAnnuallyNew * "isOrg") as "numRecurringAnnuallyNewDonationsFromOrgs",
+    SUM(recurringAnnuallyNew * (isNotRefund + isRefund)) as "numRecurringAnnuallyNewDonations",
+    SUM(recurringAnnuallyNew * (isNotRefund + isRefund) * "isUser") as "numRecurringAnnuallyNewDonationsFromUsers",
+    SUM(recurringAnnuallyNew * (isNotRefund + isRefund) * "isOrg") as "numRecurringAnnuallyNewDonationsFromOrgs",
 
         /* one-time */
-        SUM(oneTimeDonations) as "numOneTimeDonations",
-        SUM(oneTimeDonations * "isUser") as "numOneTimeDonationsFromUsers",
-        SUM(oneTimeDonations * "isOrg") as "numOneTimeDonationsFromOrgs",
+        SUM(oneTimeDonations * (isNotRefund + isRefund)) as "numOneTimeDonations",
+        SUM(oneTimeDonations * (isNotRefund + isRefund) * "isUser") as "numOneTimeDonationsFromUsers",
+        SUM(oneTimeDonations * (isNotRefund + isRefund) * "isOrg") as "numOneTimeDonationsFromOrgs",
 
     SUM(addedFunds) as "numAddedFunds",
-    SUM(totalExpensesRecorded) as "numExpensesPaid"
+    SUM(totalExpensesRecorded) as "numExpensesPaid",
+    SUM("amountInUSD" * (isRefund + isRefunded)) as "refundTransactions"
```

Comment:
Can you move this up above where all the amounts are and follow the decimal conversion? Maybe after `totalDonationsMadeOnPlatform`. That way it's easier to process when we copy paste on spreadsheet

Reply:
Sure thing! I'll do it in a min!